apps/meteor/server/features/EmailInbox/EmailInbox.ts

```diff
@@ -88,7 +92,7 @@ export async function configureEmailInboxes(): Promise<void> {
 }
 
 Meteor.startup(() => {
-	settings.watchOnce('Livechat_Routing_Method', (_) => {
+	settings.watch('Livechat_Routing_Method', (_) => {
```

Comment:
Any reason for this change? The routing method shouldn't affect how the emailinboxes are setup. The watchOnce was here because of an old race condition :eyes:

Reply:
i should probably watch a different setting, but changing this to watch allowed to change the settings and have the email inbox reload them on save